example/cordova/www/index.html

```diff
@@ -1,50 +1,111 @@
-                <p id="configArea"></p>
+<body class="color-theme-red">
+  <div id="app">
+    <!-- Status bar overlay for fullscreen mode-->
+    <div class="statusbar"></div>
+    <!-- Left panel with cover effect-->
+    <div class="panel panel-left panel-cover">
+      <div class="view">
+        <div class="page">
+          <div class="navbar">
+            <div class="navbar-inner">
+              <div class="title">AeroGear</div>
+            </div>
+          </div>
+          <div class="page-content">
+            <div class="list links-list panel-close">
+              <ul>
+                <li>
+                  <a href="/template/home/">
+                    <div class="item-media">
+                      <i class="f7-icons">home</i>
+                    </div>
+                    <div class="item-inner">
+                      Home
+                    </div>
+                  </a>
+                </li>
+                <li>
+                  <a href="/template/auth/">
+                    <div class="item-media">
+                      <i class="f7-icons">lock_fill</i>
+                    </div>
+                    <div class="item-inner">
+                      Auth
+                    </div>
+                  </a>
+                </li>
+                <li>
+                  <a href="/template/metrics/">
+                    <div class="item-media">
+                      <i class="f7-icons">graph_square</i>
+                    </div>
+                    <div class="item-inner">
+                      Metrics
+                    </div>
+                  </a>
+                </li>
+                <li>
+                  <a href="/template/push/">
+                    <div class="item-media">
+                      <i class="f7-icons">bell_fill</i>
+                    </div>
+                    <div class="item-inner">
+                      Push
+                    </div>
+                  </a>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <!-- Your main view, should have "view-main" class -->
+    <div class="view view-main ios-edges">
+      <!-- Page, data-name contains page name which can be used in callbacks -->
+      <div class="page" data-name="home">
+        <!-- Top Navbar -->
+        <div class="navbar">
+          <div class="navbar-inner">
+            <div class="left">
+              <a href="#" class="link icon-only panel-open" data-panel="left">
+                <i class="icon f7-icons ios-only">menu</i>
+                <i class="icon material-icons md-only">menu</i>
+              </a>
             </div>
+          </div>
         </div>
-        <script type="text/javascript" src="cordova.js"></script>
-        <script type="text/javascript" src="js/bundle.js"></script>
-    </body>
-</html>
+      </div>
+    </div>
+
+  </div>
+  <!-- Cordova -->
+  <!--
+  <script src="cordova.js"></script>
```

Comment:
Why this is commented? Just curious if we are aware that this is no longer used.